config/secrets.yml

```diff
@@ -2,7 +2,10 @@ shared:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 development:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
Since then are all the same, do we need the repetition in rules for each environment? or can it just use the shared keys?

Reply:
Thanks Elle, so the default: would be fine to leave as the only one?